### ISSUE TITLE
Tracker is not defined

### ISSUE THREAD

**Issue by satyavh** [maximummeteor/reactive-cursors#1]
Recently I come across this error:

```
I20150628-23:19:26.896(2)? Exception from sub id undefined ReferenceError: Tracker is not defined
I20150628-23:19:26.897(2)?     at [object Object].extendFunctions.observeChanges (packages/maximum:reactive-cursors/reactive-cursors.coffee:22:16)
I20150628-23:19:26.897(2)?     at Function.Mongo.Collection._publishCursor (packages/mongo/collection.js:303:1)
I20150628-23:19:26.898(2)?     at [object Object].Cursor._publishCursor (packages/mongo/mongo_driver.js:835:1)
I20150628-23:19:26.898(2)?     at [object Object]._.extend._runHandler (packages/ddp/livedata_server.js:986:1)
I20150628-23:19:26.898(2)?     at [object Object]._.extend._startSubscription (packages/ddp/livedata_server.js:769:1)
I20150628-23:19:26.898(2)?     at packages/ddp/livedata_server.js:407:1
I20150628-23:19:26.898(2)?     at Array.forEach (native)
I20150628-23:19:26.898(2)?     at Function._.each._.forEach (packages/underscore/underscore.js:105:1)
I20150628-23:19:26.898(2)?     at [object Object]._.extend.startUniversalSubs (packages/ddp/livedata_server.js:406:1)
I20150628-23:19:26.898(2)?     at packages/ddp/livedata_server.js:298:1
```

which makes sense, because Tracker only runs on the client. Any ideas?

**Comment by maxnowack** [comment]
This package use [peerlibrary:server-autorun](https://github.com/peerlibrary/meteor-server-autorun/) to have `Tracker` available on server.
Can you provide an meteorpad for reproduction?

**Comment by maxnowack** [comment]
should be fixed in v0.2.0
